cmd/antrea-agent/options.go

```diff
@@ -68,8 +69,10 @@ type Options struct {
 	idleFlowTimeout time.Duration
 	// Stale connection timeout to delete connections if they are not exported.
 	staleConnectionTimeout time.Duration
-	nplStartPort           int
-	nplEndPort             int
+	// IGMP query interval
```

Comment:
not a useful comment IMO, it just repeats the variable name; you could remove that line altogether

Reply:
removed